### ISSUE TITLE
Docker image `cypress/included:10.3.0` has not been published

### ISSUE THREAD

**Issue by oliverlockwood** [cypress-io/cypress#22580]
### Current behavior

```
$ docker pull cypress/included:10.3.0
Error response from daemon: manifest for cypress/included:10.3.0 not found: manifest unknown: manifest unknown
```

### Cypress Version

10.3.0

### Package Manager

other

### Operating system

Mac

### Other

https://hub.docker.com/r/cypress/included/tags shows `10.2.0` as the latest version.
https://www.npmjs.com/package/cypress shows `10.3.0` as the latest version.

Previous versions (e.g. `10.2.0`) had their corresponding `included` Docker image available within a couple of hours of the NPM artifact's publication, so I wondered if something might have gone wrong with the docker build process.

**Comment by tracymazelin** [comment]
I had the same issue yesterday but it looks like the 10.3.0 image was just published: https://hub.docker.com/layers/included/cypress/included/10.3.0/images/sha256-942468cdb722c408607093f60eeb1b4ff098a384f9123bf2ded36f55d4c96352?context=explore

**Comment by oliverlockwood** [comment]
@tracymazelin thanks, I confirm that's now working for me.  Closing this issue accordingly.
